RoaringBitmap/src/test/java/org/roaringbitmap/TestIterators.java

```diff
@@ -85,14 +84,11 @@ public void testIteration() {
     final int[] data = takeSortedAndDistinct(source, 450000);
     RoaringBitmap bitmap = RoaringBitmap.bitmapOf(data);
 
-    final List<Integer> iteratorCopy = ImmutableList.copyOf(bitmap.iterator());
```

Comment:
This makes me think one could migrate from `bitmap.iterator() `to `bitmap.stream().iterator()`

I suggest keeping the original code, and switching to this alternative way to get an `Iterable<Integer>`. Sorry for not having throught about it earlier.